agent/CLI.md

```diff
@@ -6,288 +6,298 @@
---transfer-block-size...........Number of transactions in one block to use in both money transfer loops.
---m2s-max-transactions..........Maximal number of transactions to do in money transfer loop from Main-net to S-chain(0 is unlimited).
---s2m-max-transactions..........Maximal number of transactions to do in money transfer loop from S-chain to Main-net(0 is unlimited).
---s2s-max-transactions..........Maximal number of transactions to do in money transfer loop from S-chain to S-chain(0 is unlimited).
---max-transactions..............Maximal number of transactions to do in both money transfer loops(0 is unlimited).
---m2s-await-blocks..............Maximal number of blocks to wait to appear in blockchain before transaction from Main-net to S-chain(0 is no wait).
---s2m-await-blocks..............Maximal number of blocks to wait to appear in blockchain before transaction from S-chain to Main-net(0 is no wait).
---s2s-await-blocks..............Maximal number of blocks to wait to appear in blockchain before transaction from S-chain to S-chain(0 is no wait).
---await-blocks..................Maximal number of blocks to wait to appear in blockchain before transaction between both S-chain and Main-net(0 is no wait).
---m2s-await-time................Minimal age of transaction message(in seconds) before it will be transferred from Main-net to S-chain(0 is no wait).
---s2m-await-time................Minimal age of transaction message(in seconds) before it will be transferred from S-chain to Main-net(0 is no wait).
---s2s-await-time................Minimal age of transaction message(in seconds) before it will be transferred from S-chain to S-chain(0 is no wait).
---await-time....................Minimal age of transaction message(in seconds) before it will be transferred between both S-chain and Main-net(0 is no wait).
---period........................Transfer loop period(in seconds).
---node-number=value.............S-Chain node number(0-based).
---nodes-count=value.............S-Chain nodes count.
---time-framing=value............Specifies period(in seconds) for time framing(0 to disable time framing).
---time-gap=value................Specifies gap(in seconds) before next time frame.
+```text
+    --s2s-enable....................Enables S-Chain to S-Chain transfers. Default mode. The abi-skale-manager path must be provided.
+    --s2s-disable...................Disables S-Chain to S-Chain transfers.
+    --net-rediscover=number.........SKALE NETWORK re-discovery interval(in seconds). Default is 3600 seconds or 1 hour, specify 0 to disable SKALE NETWORK re-discovery.
+```
+
+**IMA** loop can optionally use dry run, group **IMA** messages and supports various customizations:
+
+```text
+    --no-wait-s-chain...............Do not wait until S-Chain is started.
+    --max-wait-attempts=value.......Max number of S-Chain call attempts to do while it became alive and sane.
+    --skip-dry-run..................Skip dry run contract method calls.
```

Comment:
We have consistent and always present opposite options like `skip-dry-run` and `no-skip-dry-run` like many command line problems in Unix. Even of one of them is default behavior and never needed to specify. Not our invention. Default is `no-skip-dry-run` if nothing is specified.
The reason for this is to catch attention on what is invoked on command line. 
We also have `ignore-dry-run` and `no-ignore-dry-run`. Default is `ignore-dry-run` because it un-successful in some of Ethereum calls IMA does.
Having `no-skip-dry-run` and `ignore-dry-run` together allows us to see problems in logs.